src/styled.html

```diff
@@ -6,62 +6,58 @@
   <style>
-      wc-gocam-viz {
-          --panel-border-width: 1px;
-          --panel-header-background: rgba(0, 0, 0, 0.03);
-          --panel-header-padding: 0.5rem 1rem;
-          --panel-header-font-weight: normal;
-          --button-background: #0d6efd;
-          --button-color: white;
-          --button-background-hover: #0b5ed7;
-          --legend-border-width: 1px 0 0 0;
-          --legend-padding: 0.5rem 1rem;
-          --process-label-background: #7a548d;
-          --process-label-color: white;
-          --process-label-padding: 0.5rem 1rem;
-          --process-label-border-width: 0;
-          --activity-border-color: #d1b6df;
-          --activity-border-width: 0 0 3px 0;
-          --activity-background-active: #af732a;
-          --activity-color-active: #eee;
-          --gene-product-color: #333;
-          --gene-product-padding: 0.5rem 1rem;
-          --gene-product-border-width: 0 0 1px 0;
-          --function-label-padding: 0.5rem 1rem;
-          --function-nodes-padding: 0px 1rem 0.5rem 1rem;
-          --node-border-width: 0 0 1px 0;
-          --graph-padding: 1rem;
-      }
+    wc-gocam-viz {
+      --panel-border-width: 1px;
+      --panel-header-background: rgba(0, 0, 0, 0.03);
+      --panel-header-padding: 0.5rem 1rem;
+      --panel-header-font-weight: normal;
+      --button-background: #0d6efd;
+      --button-color: white;
+      --button-background-hover: #0b5ed7;
+      --legend-border-width: 1px 0 0 0;
+      --legend-padding: 0.5rem 1rem;
+      --process-label-background: #7a548d;
+      --process-label-color: white;
+      --process-label-padding: 0.5rem 1rem;
+      --process-label-border-width: 0;
+      --activity-border-color: #d1b6df;
+      --activity-border-width: 0 0 3px 0;
+      --activity-background-active: #af732a;
+      --activity-color-active: #eee;
+      --gene-product-color: #333;
+      --gene-product-padding: 0.5rem 1rem;
+      --gene-product-border-width: 0 0 1px 0;
+      --function-label-padding: 0.5rem 1rem;
+      --function-nodes-padding: 0px 1rem 0.5rem 1rem;
+      --node-border-width: 0 0 1px 0;
+      --graph-padding: 1rem;
+    }
 
-      wc-gocam-viz::part(gene-product) {
-          font-size: 1.125rem;
-          font-weight: bold;
-      }
+    wc-gocam-viz::part(gene-product) {
+      font-size: 1.125rem;
+      font-weight: bold;
+    }
 
-      wc-gocam-viz::part(legend-header) {
-          text-align: center;
-      }
+    wc-gocam-viz::part(legend-header) {
+      text-align: center;
+    }
 
-      wc-gocam-viz::part(gocam-title) {
-          font-weight: bold;
-      }
+    wc-gocam-viz::part(gocam-title) {
+      font-weight: bold;
+    }
   </style>
 </head>
 
 
 <body>
 
   <div style='max-width: 1200px; margin: 0 auto'>
-    <wc-gocam-viz
-      id='gocam-1'
-      gocam-id='568b0f9600000284'
-      show-legend='true'
-    >
+    <wc-gocam-viz id='gocam-1' api-url='http://barista-dev.berkeleybop.org/search//stored?id=%ID'
```

Comment:
Setting the `api-url` attribute here is fine locally if that's needed for your testing, but that change shouldn't be committed.

Reply:
yes, I have reverted the styles.html